gno.land/pkg/integration/testing_integration.go

```diff
@@ -273,3 +338,34 @@ func tsValidateError(ts *testscript.TestScript, cmd string, neg bool, err error)
 		}
 	}
 }
+
+type envSetter interface {
+	Setenv(key, value string)
+}
+
+// createAccount creates a new account with the given name and adds it to the keybase.
+func createAccount(env envSetter, kb keys.Keybase, accountName string) (balance gnoland.Balance, err error) {
```

Comment:
What do you think about renaming this method?

It's called `createAccount`, but returns a `gnoland.Balance`. Also, please consider dropping named returns

Reply:
I think it's okay; I named it `createAccount` because it creates an account.  Then it returns the balance of the account that was created.

I agree with you on the named returns; I'll remove them.

Reply:
Resolved as part of https://github.com/gnolang/gno/pull/1471/commits/05ceef4cdf9ecf535ac2d661488cb5e7680dde8f